commons/src/main/java/org/dhis2/commons/bindings/CommonExtensions.kt

```diff
@@ -0,0 +1,89 @@
+package org.dhis2.commons.bindings
+
+import android.content.res.Resources
+import android.graphics.Color
+import android.graphics.PorterDuff
+import android.graphics.Typeface
+import android.text.method.ScrollingMovementMethod
+import android.util.TypedValue
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.TextView
+import androidx.annotation.DrawableRes
+import androidx.databinding.BindingAdapter
+import org.dhis2.commons.R
+import java.util.ArrayList
+import kotlin.math.pow
+
+@BindingAdapter("scrollingTextView")
```

Comment:
I would like to switch the way we use extensions, i think that we could have split classes for different scopes maybe by  Android component (RecyclerViewBindings.kt...) or by DHIS2 component (FormBindings.kt...) what do you think?